### PR TITLE
fix: issue 19395 change delete icon color in dark model

### PR DESCRIPTION
**Pull request by sxt90128**
Issue: https://github.com/argoproj/argo-cd/issues/19395

In Darkmode, when edit application, the color for delete icon is not obvious and consistent

To Reproduce
Enable "Dark mode", go to an application, click "DETAILS", in SUMMARY tab, click "EDIT", the delete icon for "ANNOTATIONS" is colored in black, which is not visible and inconsistent.

Result after fix:
image (13)

### CODE REVIEW

**Comment by sxt90128** [comment]
@wanghong230 , @pasha-codefresh , @ishitasequeira , @todaywasawesome , @christianh814 
Hi all, I have created a PR the fix that issue of icon color in dark model, could you help to review it, thanks.

**Review by gdsoumya — DISMISSED** [review]

File: ui/src/app/applications/components/application-summary/application-summary.scss [hunk 1]
```diff
@@ -41,50 +36,46 @@
...
             padding-right: 10px;
         }
-
         .argo-form-row__error-msg {
             position: static;
             line-height: 1;
         }
     }
-}
+}
```
[the remark this answers is not in the document]
Comment (sxt90128) [reply ui/src/app/applications/components/application-summary/application-summary.scss:81]: Sure, will do

[the remark this answers is not in the document]
Comment (todaywasawesome) [reply ui/src/app/applications/components/application-summary/application-summary.scss:81]: @sxt90128 Actually can you revert the other ones? There are quite a few spaces removed.

**Review by todaywasawesome — CHANGES_REQUESTED** [review]

File: ui/src/app/applications/components/application-summary/application-summary.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (sxt90128) [reply ui/src/app/applications/components/application-summary/application-summary.scss]: @gdsoumya @todaywasawesome I have reverted all the formatting changes, please review again, thanks

**Review by ishitasequeira — APPROVED** [review]
LGTM!

**Review by todaywasawesome — APPROVED** [review]
Excellent thanks!

**Review by pasha-codefresh — APPROVED** [review]

**Review by christianh814 — APPROVED** [review]

**Pull request merged**